shared/src/Datadog.Trace.ClrProfiler.Native/EnvironmentVariables.h

```diff
@@ -11,4 +11,6 @@ class EnvironmentVariables final
     inline static const shared::WSTRING LogPath = WStr("DD_TRACE_LOG_PATH");
     inline static const shared::WSTRING LogDirectory = WStr("DD_TRACE_LOG_DIRECTORY");
     inline static const shared::WSTRING DebugLogEnabled = WStr("DD_TRACE_DEBUG");
+    inline static const shared::WSTRING IncludeProcessNames = WStr("DD_PROFILER_PROCESSES");
+    inline static const shared::WSTRING ExcludeProcessNames = WStr("DD_PROFILER_EXCLUDE_PROCESSES");
```

Comment:
If these were never documented, it might be a good time to rename them to drop the "PROFILER" in the name? Or maybe we can support both a new name and the old for backwards compat?

Reply:
> If these were never documented, it might be a good time to rename them to drop the "PROFILER" in the name

I'll leave this for a separate PR, but it's more hassle than it's worth IMO